src/main/java/pl/confitura/jelatyna/admin/Repository.java

```diff
@@ -0,0 +1,11 @@
+package pl.confitura.jelatyna.admin;
+
+import java.util.List;
+
+@org.springframework.stereotype.Repository("adminRepository")
+public interface Repository extends org.springframework.data.repository.Repository<Admin, Long> {
```

Comment:
AD 3. jeśli rezygnujemy z data-rest to chyba nie ma znaczenia jeśli będzie więcej metod?